### ISSUE TITLE
Tasks Runner waits for external tool (GIT Extension) to be closed after launching it

### ISSUE THREAD

**Issue by Aukstkalnis** [Microsoft/vscode#19836]
- VSCode Version: 1.9
- OS Version: win7x64

Steps to Reproduce:

1. Configure new task:
{
    "taskName": "Open GIT Extesion",
    "suppressTaskName": true,
    "isShellCommand": true,
    "command":"gitex"
}
2. VS Code Task runner waits for GIT Extension to be closed and then shows that task was complete

Why it is the problem? Because when I launch it from terminal, command is done immediately and there is no waiting, so why task runner is waiting?

Updated:
When I execute another task, it asks to terminate running task and when I press terminate button it throws error: "The launched process doesn't exist anymore. If the task spawned background tasks exiting VS Code might result in orphaned processes."

**Comment by dbaeumer** [comment]
The problem is that the gitex launches a second process which inherits the stdio streams from gitex. As a result the streams are open and we think that the process is still running (which in fact is the case). 

This will improve with the work we do to run task in terminal which will allow to run more than one in parallel and handle this better. I will close the issue since there is no work planned to fix this on the old task output runner.
